### DISCUSSION TITLE
AMD Radeon Support for coqui TTS

### DISCUSSION THREAD

**Discussion by SadeghKrmi** [coqui-ai/TTS#2904] [General Q&A]
I am wondering if there is possibility to use AMD Radeon RX 580 GPU's to train a new model or it has to be Nvidia GPU?

**Comment by Milor123** [comment]
Maybe using ROCm in linux, it work for me in m y Rx 6750XT but i am using ROCm. But i recommend you buy NVDIA card, this shits of AMD are unuseful for IA, all is very hard and work with many bugs :/ AMD is a crappy a 3050ti works better than my card in IA work, and you are in windows with AMD you are die xDDD.

PD: The new ROCm for windows, is only SDK dont work for IA, surely it take a few years for working but surely with many bugs like in linux.

**Reply by maxime-fleury** [reply]
Update: AMD finnaly fully open sourced ROCm, so maybe we will have coqui tts to run on windows on AMD GPU "one day"

source: https://wccftech.com/amd-rocm-going-open-source-will-include-software-stack-hardware-documentation/

**Reply by aelfwyne** [reply]
AMD now has hardware AI cores and has improved RocM support on both Linux and Windows. I wonder what excuse there is now to not support it?

**Reply by eginhard** [reply]
Pytorch is available now for AMD/ROCm: https://pytorch.org/get-started/locally/

As long as you install the correct Pytorch yourself it should probably work. You should also use [our Coqui fork](https://github.com/idiap/coqui-ai-TTS) (available via `pip install coqui-tts`) to be able to use recent versions of Python, Pytorch, etc. This repo is not maintained anymore. Feel free to open an issue in the fork if necessary, but usually issues would be on the Pytorch, not Coqui side.

**Comment by Milor123** [comment]
> Update: AMD finnaly fully open sourced ROCm, so maybe we will have coqui tts to run on windows on AMD GPU "one day"
> 
> source: https://wccftech.com/amd-rocm-going-open-source-will-include-software-stack-hardware-documentation/

Ohhh xD, Now I've sell the AMD GPU, and now have a NVIDIA RTX hahah, i've solved all my problems. I hate AMD NOW xD
